Gemfile.lock

```diff
@@ -1,6 +1,7 @@
 GIT
   remote: https://github.com/projectblacklight/blacklight.git
-  revision: 94749d5aa34c4474f4ad27c1dc35f95442271f23
+  revision: d3e9eb1977ca81e6d5737870524a3466dd29ad93
+  branch: avoid_unnecessary_lookup
```

Comment:
```
li-dl-7328-02d5:rialto-webapp jlittman$ bundle install
Fetching gem metadata from https://rubygems.org/.......
Fetching https://github.com/projectblacklight/blacklight.git
fatal: Could not parse object 'd3e9eb1977ca81e6d5737870524a3466dd29ad93'.
Revision d3e9eb1977ca81e6d5737870524a3466dd29ad93 does not exist in the repository https://github.com/projectblacklight/blacklight.git. Maybe you
misspelled it?
```

Reply:
I resolved this with a bundle update. That branch got rebased.